pkg/aws/eni/routing/routing.go

```diff
@@ -122,6 +129,73 @@ func Install(ip net.IP, info *RoutingInfo, mtu int, masq bool) error {
 	return nil
 }
 
+// Delete removes the ingress and egress rules that control traffic for
+// endpoints. Note that the routes within these rules are not deleted as they
+// can be reused when another endpoint is created on the same node. The reason
+// for this is that each ENI device has an index called ifindex which is used
+// as the table ID when these rules are created. The routes are created inside
+// a table with this ID, and because this table ID is tied to the ENI device,
+// it's stable to rely on and therefore can be reused.
+func Delete(ip net.IP) error {
+	ipWithMask := net.IPNet{
+		IP:   ip,
+		Mask: net.CIDRMask(32, 32),
+	}
+
+	scopedLog := log.WithFields(logrus.Fields{
+		"ip": ipWithMask.String(),
+	})
+
+	// Ingress rules
+	ingress := route.Rule{
+		Priority: 20, // After encryption and proxy rules, before local table
```

Comment:
Hmm, I wonder where we could centralize the definition of this. We should be referring to rule priority from several places, obviously here in ENI, but also as you mention in encryption and proxy. Maybe we need a `pkg/datapath/linux/constants` or similar?

Reply:
(that said it's also possible that this is the first instance in the Go side where we actually specify the rule priority, where previously we've only had these in `bpf/init.sh`.)

Reply:
Good point; moved them to `pkg/datapath/linux/linux_defaults` package.